stdpopsim/qc/arabidopsis_thaliana_qc.py

```diff
@@ -61,6 +66,10 @@ def __init__(self):
         self.migration_matrix = [[0]]
 
 
+_species.get_demographic_model(

```

Comment:
How about 
```python
_species.get_demographic_model( "SouthMiddleAtlas_1D17").register_qc(Durvasula2017MSMC())
```
There's a tiny bit less typing and it seems redundant to type ``qc_model`` when it's a method of the DemographicModel class, and we're associating the QC with it.

Reply:
Done!